packages/addons/service/locale/source/resources/language/English/strings.po

```diff
@@ -0,0 +1,23 @@
+# Kodi Media Center language file
+msgid ""
+msgstr ""
+
+msgctxt "#30000"
+msgid "Configuration"
+msgstr ""
+
+msgctxt "#30001"
+msgid "Charmap"
+msgstr ""
+
+msgctxt "#30002"
+msgid "Locale"
+msgstr ""
+
+msgctxt "#30003"
+msgid "LANG has changed from {} to {}."
```

Comment:
You can put "Locale changed from {} to {}. Please reboot to apply globally." in a single msgid. I'm not sure there's a reason to split the statements into two.